test/engine/update.result

```diff
@@ -1135,3 +1135,27 @@ s:update({1}, {{'!', 'field4.key1', 0}})
 ---
 - error: Field 'field4.key1' was not found in the tuple
 ...
+s:drop()
+---
+...
+-- Arrays
+s = box.schema.create_space('test', {engine = engine})
+---
```

Comment:
As I told, there is no need for a space. But since you created it, you also need to drop it in the end of the test.